README.md

```diff
@@ -120,6 +124,16 @@ Go ahead and try something like:
 
 When you have an account like this, 'Assets:Bank:Checking' should show up.
 
+If fuzzy matching based account completion is enabled, the matches are
+loaded based on string similarity and without regars for the sub-levels.
```

Comment:
```suggestion
loaded based on string similarity and without regard for the sub-levels.
```

Reply:
Thanks for catching this. I have fixed the typo.